src/client/Microsoft.Identity.Client/Internal/Broker/NullBroker.cs

```diff
@@ -23,5 +23,10 @@ public Task<MsalTokenResponse> AcquireTokenUsingBrokerAsync(Dictionary<string, s
         {
             throw new PlatformNotSupportedException(MsalErrorMessage.BrokerNotSupportedOnThisPlatform);
         }
+
+        public Task<IEnumerable<IAccount>> GetAccountsAsync(string clientID)
```

Comment:
can this be unit tested?

Reply:
yes

Reply:
Yes, we added a unit test showing that GetAccounts calls this method.